packages/wonder-blocks-button/components/button.js

```diff
@@ -1,52 +1,68 @@
+     *
+     * Note: Either href or onClick must be defined
+     */
+    href?: string,
+
+    /**
+     * Whether to use client-side navigation.
+     *
+     * If the URL passed to href is local to the client-side, e.g.
+     * /math/algebra/eval-exprs, then use ReactRouter to do a client side
+     * navigation by doing history.push(this.props.href) using
+     * ReactRouter's history object
+     */
+    clientSideNav?: boolean,
+
+    /**
+     * Function to call when button is clicked.
+     *
+     * This callback should be used for things like marking BigBingo
+     * conversions. It should NOT be used to redirect to a different URL or to
+     * prevent navigation via e.preventDefault(). The event passed to this
+     * handler will have its preventDefault() and stopPropagation() methods
+     * stubbed out.
+     *
+     * Note: onClick is optional if href is present, but must be defined if
+     * href is not
+     */
     onClick?: (e: SyntheticEvent<MouseEvent | TouchEvent>) => void,
 };
 
 export default class Button extends React.Component<Props> {
     render() {
         return (
-            <a
-                className={css(
-                    styles.button,
-                    styles[this.props.color || "blue"],
-                )}
-                onClick={this.props.onClick}
+            <div
+                onClick={this.props.onClick && this.props.onClick}
+                href={this.props.href && this.props.href}
             >
-                <LabelLarge>Hello, world!</LabelLarge>
-            </a>
+                <ButtonCore
+                    icon={this.props.icon ? this.props.icon : undefined}
+                    // spinner={this.props.spinner} // TODO
+                    color={this.props.color ? this.props.color : undefined}
+                    kind={this.props.kind ? this.props.kind : undefined}
+                    light={this.props.light ? this.props.light : undefined}
+                    size={this.props.size ? this.props.size : undefined}
+                    disabled={
+                        this.props.disabled ? this.props.disabled : undefined
+                    }
+                    testId={this.props.testId}
+                    style={this.props.style ? this.props.style : undefined}
+                >
+                    {this.props.children}
+                </ButtonCore>
+            </div>
         );
     }
 }
 
-const styles = StyleSheet.create({
-    button: {
-        display: "inline-block",
-        minHeight: 40,
-        maxHeight: 40,
-        color: Color.white,
-        padding: "10px 16px",
-        boxSizing: "border-box",
-        borderRadius: 4,
-        cursor: "pointer",
-    },
-    blue: {
-        backgroundColor: Color.blue,
-    },
-    green: {
-        backgroundColor: Color.green,
-    },
-    gold: {
-        backgroundColor: Color.gold,
-    },
-    red: {
-        backgroundColor: Color.red,
-    },
-});
+const styles = StyleSheet.create({});
```

Comment:
I wonder if Button will never have any styling of its own? It doesn't feel like it would!